we_get/core/module.py

```diff
@@ -20,22 +19,19 @@ def http_get_request(self, url):
         """ http_request: create HTTP request.
           @return: data.
         """
-        opener = urllib.request.build_opener()
-        opener.addheaders = [('User-Agent', USER_AGENT), ("Accept", "*/*")]
-        try:
-            res = opener.open(url).read().decode()
-            return res
-        except URLError as err:
+        headers = [('User-Agent', USER_AGENT), ("Accept", "*/*")]
+        res = requests.get(url, headers)
```

Comment:
https://docs.python-requests.org/en/latest/api/#requests.request

based on that headers must be dict so it should be

```python
res = requests.get(url, headers=headers)
```